Good-Bids/goodbids
# [#96] Bid Button

# Summary

- Adds clsx
- Adds bid button

## Issues

- #96

## Testing Instructions

- Start WP (`bin/start`) and Vite (`bin/start-vite`)
- Create an auction
- Visit said auction
- See the bid button (you can even click it)

Presently timer inputs are not dynamic (coming in #140), so you will need to manually update the `data-initial` values in `client-mu-plugins/goodbids/blocks/bid-now/render.php` to see all of the states.

## Screenshots

![image](https://github.com/Good-Bids/goodbids/assets/48295174/827020a0-efec-4321-980e-97678996d6b8)
